### DISCUSSION TITLE
Next step options

### DISCUSSION THREAD

**Discussion by drizzt09** [owncast/owncast#3776] [Q&A]
I have just installed owncast on Ubuntu.

I am wondering if there is a way to either embed the stream in discord channel without hitting discord stream user limit in audio channel. But still use the chat feature in discord along with bots instead of free chat or federated chat.

If not is there a way to maybe embed a FB community chat in owncast?

Or can bots be used with federated chat? I would rather bots work with switches like in discord within the chat. Not on the broadcast. 

Lastly, are there tags that can be used like tied to the stream key name? I have multiple streamers for different times. I am admin but create each streamer own stream key with streamers name as name for key. Would like to use a tag to display who is treating based on the key being used. Feeding off the tag idea. In the admin console at the top is 'what are you streaming' but only admin can do that. Would nice if the streamer could do that via tag too. 

Any thoughts or am I way outside the the bubble.

**Comment by gabek** [comment]
Unfortunately, I don't know much about how Discord or Facebook works, so I can't tell you what its available feature set is, and if Owncast can be used with it.

You can build bots for Owncast. This page is a good starting point: https://owncast.online/thirdparty/.

As for the streamer being able to change the tags and stream title, Owncast is built to be single-user. So it's assumed that the streamer and the admin are the same person. I know some people don't follow this model, like in your case, and that's great. But this is how Owncast is designed to be used, and why there's no concept of user management for logging into the admin. It's a part of what keeps Owncast simple and approachable. But it does mean if you're going to let others use your server, you'll have to manage it for them.

**Comment by drizzt09** [comment]
If it is only for 1 streamer, then what is the point of the ability to have
multiple keys?

**Reply by gabek** [reply]
Whatever makes sense to you. It never used to have multiple keys, but [people asked for it](https://github.com/owncast/owncast/issues/2320), so now it does.
